### PR TITLE
[EN-3836] deprecate validation classes

### PR DESCRIPTION
**Pull request by SirenaBorracha**
## Description

This PR removes deprecated validation code.

## Checklist for Reviewer

- [ ] Review code changes
- [ ] Review changes for Database effects
- [ ] Verify change works in IE browser

More details about this can be found in [docs/review.md](docs/review.md)

### CODE REVIEW

File: src/validators/policeoffenses.js [hunk 1]
```diff
@@ -13,30 +10,4 @@ export const validatePoliceOffenses = (data, formType, options = {}) => {
...
-      checkValue(attributes.HasOffenses, 'Yes')
-        ? {
-          presence: true,
-          accordion: { validator: offense },
-        } : {}
-    ),
-  }
-
-  return validateModel(data, policeOffensesModel, { ...options, ...modelOptions })
-}
```
Comment (suzubara) [inline src/validators/policeoffenses.js]: The above is part of the `validatePoliceOffenses` function and shouldn't have been deleted

File: src/validators/policeotheroffenses.js [hunk 2]
```diff
@@ -1,38 +1,9 @@
...
-        ? {
-          presence: true,
-          accordion: { validator: otherOffense },
-        } : {}
-    ),
-  }
-
-  return validateModel(data, policeOtherOffensesModel, { ...options, ...modelOptions })
-}
-
```
Comment (suzubara) [inline src/validators/policeotheroffenses.js]: same comment, the code in `validatePoliceOtherOffenses` shouldn't be deleted

File: src/validators/psychological.js [hunk 3]
```diff
@@ -7,11 +7,11 @@ export default class PsychologicalValidator {
   }
```
Comment (suzubara) [inline src/validators/psychological.js]: Can we delete `class PsychologicalValidator`?

Comment (SirenaBorracha) [reply src/validators/psychological.js]: It seems integral to `showQuestion21E` and `hideExistingConditions` in the same file, and those two show up in a lot of different places.

File: src/validators/policeoffenses.js (hunk 1 above)
Comment (SirenaBorracha) [reply src/validators/policeoffenses.js]: Fixed

File: src/validators/policeotheroffenses.js (hunk 2 above)
Comment (SirenaBorracha) [reply src/validators/policeotheroffenses.js]: Fixed

File: src/validators/psychological.js (hunk 3 above)
Comment (SirenaBorracha) [reply src/validators/psychological.js]: Fixed

**Pull request merged**